### PR TITLE
Fix production color label

### PR DESCRIPTION
**Pull request by wildeyedskies**
Move away from chick-fil-a branding.

### CODE REVIEW

**Comment by shellderp** [comment]
why is this being changed? the graphics were great

**Review by keeferrourke — APPROVED** [review]
Love the happy creature :)

**Review by shellderp — CHANGES_REQUESTED** [review]
Please make it configurable if you really want this, or I'll have to fork the repo

**Comment by keeferrourke** [comment]
> why is this being changed? the graphics were great

The graphics may have been great, but so are the new ones! Association with Chick-fil-a is not really desirable, considering the brand's exclusionary and [anti-LGBT history](https://en.wikipedia.org/wiki/Chick-fil-A_and_LGBT_people).

File: README.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 Backfila is a service that manages backfill state, calling into other services to do batched work.
 
+Note: some instructions here are out of date
```
Comment (mpawliszyn) [inline README.md:5]: Which instructions? It is quite possible.

**Review by escardin — APPROVED** [review]

**Comment by shellderp** [comment]
thanks, can you make the favicon a 🅱️ ? then it's distinguishable from other cash sites. Would be good to run local dev and make sure it renders ok

**Comment by wildeyedskies** [comment]
> thanks, can you make the favicon a 🅱️ ? then it's distinguishable from other cash sites. Would be good to run local dev and make sure it renders ok

Yes, done now!

**Comment by shellderp** [comment]
I think you missed one favicon? There's two 🤔

**Comment by wildeyedskies** [comment]
> I think you missed one favicon? There's two 🤔

Gotcha, I changed out the second one

**Review by shellderp — APPROVED** [review]

**Pull request merged**